lib/components/shorts/short_scroll_view.dart

```diff
@@ -203,22 +194,18 @@ class ShortScrollView extends HookConsumerWidget {
       if (currentIndex.value != index || !context.mounted) return;
 
       if (preloadNext) {
-        Future.delayed(500.ms, () {
```

Comment:
Hmm tror jeg tenkte at det var lurt for å unngå at preloading påvirket oppstart av current.
Var nok premature optimization